src/app/components/common/dialog/jobs-manager/components/job-item/job-item.component.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Component, ChangeDetectionStrategy, Input, Output, EventEmitter,
+} from '@angular/core';
+import { untilDestroyed, UntilDestroy } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { filter } from 'rxjs/operators';
+import { JobState } from 'app/enums/job-state.enum';
+import { ApiTimestamp } from 'app/interfaces/api-date.interface';
+import { Job } from 'app/interfaces/job.interface';
+import { DialogService } from 'app/services';
+import { LocaleService } from 'app/services/locale.service';
+import { WebSocketService } from 'app/services/ws.service';
+import { T } from 'app/translate-marker';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-job-item',
+  templateUrl: './job-item.component.html',
+  styleUrls: ['./job-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class JobItemComponent {
+  @Input() job: Job;
+  @Output() aborted = new EventEmitter();
+  readonly JobState = JobState;
+
+  constructor(
+    private ws: WebSocketService,
+    private localeService: LocaleService,
+    private dialogService: DialogService,
+    private translate: TranslateService,
+  ) {}
+
+  getReadableDate(input: ApiTimestamp): string {
```

Comment:
Okay for now, but in the future this will need to be a pipe. Please add a TODO comment, so that we don't forget.